Add multi-symbol handling and prevent day trading

Kyle-Lin1 opened the pull request:
You can now trade as many symbols as you want by adding or removing the symbols to the list  "symbols". Ensure you are doing that on lines 111 and 121 (you should change it on line 26 to be safe but it doesn't matter). 

You can also change self.sleeptime to whatever you want and not worry about day trading. My testing found that 24H is still one of the better general self.sleeptime, but knowing you can't day trade is still nice. If you want to day trade you can just comment out lines 68-72, 88, and 102 (the most important lines are 70-72).

Kyle-Lin1 committed:
Add multi-symbol handling and prevent day trading

Kyle-Lin1 committed:
Add multi-symbol handeling and prevent day trading

Kyle-Lin1 committed:
Merge branch 'new_branch' of https://github.com/Flawlesssharky6/MLTradingBot into new_branch

Kyle-Lin1 commented:
I am trying to add another strategy to synergize with the sentiment one that is currently in use because I feel like the results can be improved (ex: the max drop downs). I am currently trying to use trend following with adaptive position sizing (so far it seems too strict to make trades and significantly improve results) and breakouts (working out some kinks and I'm not sure if it will significantly improve the results). I am still working on these but does anyone have some tips?

Flawlesssharky6 commented:
I am trying to add another strategy to synergize with the sentiment one that is currently in use because I feel like the results can be improved (ex: the max drop downs). I am currently trying to use trend following with adaptive position sizing (so far it seems too strict to make trades and significantly improve results) and breakouts (working out some kinks and I'm not sure if it will significantly improve the results). I am still working on these but does anyone have some tips?

Flawlesssharky6 commented:
I am trying to add another strategy to synergize with the sentiment one that is currently in use because I feel like the results can be improved (ex: the max drop downs). I am currently trying to use trend following with adaptive position sizing (so far it seems too strict to make trades and significantly improve results) and breakouts (working out some kinks and I'm not sure if it will significantly improve the results). I am still working on these but does anyone have some tips?

Kyle-Lin1 committed:
added check-day-trades

Kyle-Lin1 commented:
I just added a program to check for day trades (I just imported a project on my repository as a folder into this project). I am confident that the program will work. But if the time conversions fail the trades are extracted and logged for you to review. 

I'm unfamiliar with licenses but I added one so I don't get in trouble. But if you encounter problems, please contact me or leave a comment.
